### PR TITLE
Pioneer12b Glider Updates and gi_363 recovery

### PR DESCRIPTION
**Pull request by cdobs**
Cruises:
-WJ20190617 Pioneer 12b
-AR35-03 Niskine cruise (gi_363 recovery)

Recovered:
-CP05MOAS-GL387 D7
-GI05MOAS-GL363 D3

Deployed:
-CP05MOAS-GL388 D8
-CP05MOAS-GL379 D7
-CP05MOAS-GL376 D10

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -1,164 +1,166 @@
...
+ML20190417,R/V Miss Linda,2019-04-07T15:07:00,2019-04-17T22:00:00,Endurance glider ce_327 recovery
+SKQ201910S,R/V Sikuliaq,2019-04-19T20:00:00,2019-04-28T15:46:47,Endurance 11A
+OC1905C,R/V Oceanus,2019-05-03T20:00:00,2019-05-08T00:00:00,Endurance 11B
+BK20190504,M/V Blackhawk,2019-05-04T09:00:00,2019-05-04T20:00:00,Pioneer cp_340 recovery
+SKQ201913T,R/V Sikuliaq,2019-05-14T00:00:00,2019-05-27T00:00:00,Papa Glider gp_469 recovery on the San Diego - Seward transit
+TS20190516,F/V Tenacious II,2019-05-16T12:00:00,2019-05-17T12:00:00,Pioneer 12a glider deployment
+AR35-03,R/V Neil Armstrong,2019-05-27T00:00:00,2019-06-20T23:59:59,Irminger Glider gi_363 Recovery - Niskine Process Cruise
+EK20190529,R/V Elakha,2019-05-29T16:22:00,2019-05-29T19:33:00,ce01issp_00013 deployment
+AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
+WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv]: looks good.

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
+P10b,,,,CP05MOAS-GL376-05-PARADM000,9,1,2018-07-24T15:52:09,2018-10-01T13:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50153,39.90575,-70.6454,,1000,1000,SS-1 Recovered by Eric Hess due to damaged rudder
+WJ20190617,,,,CP05MOAS-GL376-00-ENG000000,10,1,2019-06-18T23:36:35,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.91363,-70.9107,,,,FZ-2
+WJ20190617,,,,CP05MOAS-GL376-01-ADCPAM000,10,1,2019-06-18T23:36:35,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-00545,39.91363,-70.9107,,,,FZ-2
+WJ20190617,,,,CP05MOAS-GL376-02-FLORTM000,10,1,2019-06-18T23:36:35,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03921,39.91363,-70.9107,,,,FZ-2
+WJ20190617,,,,CP05MOAS-GL376-03-CTDGVM000,10,1,2019-06-18T23:36:35,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09029,39.91363,-70.9107,,,,FZ-2
+WJ20190617,,,,CP05MOAS-GL376-04-DOSTAM000,10,1,2019-06-18T23:36:35,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00364,39.91363,-70.9107,,,,FZ-2
+WJ20190617,,,,CP05MOAS-GL376-05-PARADM000,10,1,2019-06-18T23:36:35,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50145,39.91363,-70.9107,,,,FZ-2
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL376_Deploy.csv]: verified glider sensor s/ns via FCP 3304-00150-00109.
verified deploy date/time & location via deploy record 3304-00021-00088.
Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 3]
```diff
@@ -34,4 +34,10 @@ TS20190205,,TS20190516,,CP05MOAS-GL379-01-ADCPAM000,6,1,2019-02-05T07:54:00,2019
...
\ No newline at end of file
+TS20190205,,TS20190516,,CP05MOAS-GL379-05-PARADM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,39.9166,-70.36972222,,1000,1000,SS-1
+WJ20190617,,,,CP05MOAS-GL379-00-ENG000000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-01-ADCPAM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-03-CTDGVM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-04-DOSTAM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL379_Deploy.csv]: Deploy date according to deploy doc 3304-00021-00087 is 2019-06-18 - needs to be corrected in deploy csv (time is correct). Location is correct.
Verified glider s/ns via FCP 3304-00021-00087 (see FCP autoexec for ADCPAM s/n)

File: deployment/CP05MOAS-GL387_Deploy.csv [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
+AR-34,,WJ20190617,,CP05MOAS-GL387-00-ENG000000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,39.9681,-71.00002,,1000,1000,FZ-1
+AR-34,,WJ20190617,,CP05MOAS-GL387-01-ADCPAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-00570,39.9681,-71.00002,,1000,1000,FZ-1
+AR-34,,WJ20190617,,CP05MOAS-GL387-02-FLORTM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04465,39.9681,-71.00002,,1000,1000,FZ-1
+AR-34,,WJ20190617,,CP05MOAS-GL387-03-CTDGVM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09027,39.9681,-71.00002,,1000,1000,FZ-1
+AR-34,,WJ20190617,,CP05MOAS-GL387-04-DOSTAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00569,39.9681,-71.00002,,1000,1000,FZ-1
+AR-34,,WJ20190617,,CP05MOAS-GL387-05-PARADM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50146,39.9681,-71.00002,,1000,1000,FZ-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL387_Deploy.csv]: Looks good.

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 5]
```diff
@@ -41,3 +41,9 @@ AR-31,,WJ20190203,,CP05MOAS-GL388-02-FLORTM000,7,1,2018-10-30T22:11:00,2019-02-0
...
 AR-31,,WJ20190203,,CP05MOAS-GL388-05-PARADM000,7,1,2018-10-30T22:11:00,2019-02-04T00:30:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50163,40.02455,-70.70045,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL388-00-ENG000000,8,1,2019-06-18T22:58:15,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,,,CP05MOAS-GL388-01-ADCPAM000,8,1,2019-06-18T22:58:15,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-00537,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,,,CP05MOAS-GL388-02-FLORTM000,8,1,2019-06-18T22:58:15,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03190,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,,,CP05MOAS-GL388-03-CTDGVM000,8,1,2019-06-18T22:58:15,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09083,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,,,CP05MOAS-GL388-04-DOSTAM000,8,1,2019-06-18T22:58:15,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00170,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,,,CP05MOAS-GL388-05-PARADM000,8,1,2019-06-18T22:58:15,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50203,39.92227,-70.91603,,1000,1000,SS-2
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL388_Deploy.csv]: Verified glider sensor s/ns via FCP 3304-00150-00110.
Verified glider deployment date, time & location via deploy doc 3304-00021-00086.
Looks good.

File: deployment/GI05MOAS-GL363_Deploy.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
...
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+AR30-03,,AR35-03,,GI05MOAS-GL363-00-ENG000000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location
+AR30-03,,AR35-03,,GI05MOAS-GL363-01-FLORDM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location
+AR30-03,,AR35-03,,GI05MOAS-GL363-02-DOSTAM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location
+AR30-03,,AR35-03,,GI05MOAS-GL363-04-CTDGVM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-09065,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL363_Deploy.csv]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 3 above)
Comment (cdobs) [reply deployment/CP05MOAS-GL379_Deploy.csv]: Fixed

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 7]
```diff
@@ -35,9 +35,9 @@ TS20190205,,TS20190516,,CP05MOAS-GL379-02-FLORTM000,6,1,2019-02-05T07:54:00,2019
...
-WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1
\ No newline at end of file
+WJ20190617,,,,CP05MOAS-GL379-00-ENG000000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-01-ADCPAM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-03-CTDGVM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-04-DOSTAM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL379_Deploy.csv]: Date correction looks good.

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv]: Looks good.

File: deployment/CP05MOAS-GL376_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL376_Deploy.csv]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 7 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL379_Deploy.csv]: Should the ADCP (DVL) be CGINS-ADCPAM-49982 or CGINS-ADCPAM-00548?

File: deployment/CP05MOAS-GL388_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL388_Deploy.csv]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 7 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL379_Deploy.csv]: I added in the notes column that the UID for DVL will need to be changed from CGINS-ADCPAM-49982 to CGINS-ADCPAM-00548

File: deployment/CP05MOAS-GL387_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL387_Deploy.csv]: Looks good.

File: deployment/GI05MOAS-GL363_Deploy.csv (hunk 6 above)
Comment (snwhite) [reply deployment/GI05MOAS-GL363_Deploy.csv]: Looks good.

**Pull request merged**